index.js

```diff
@@ -46,6 +46,41 @@ function runNightwatch (done) {
 
 	try {
 		Nightwatch.cli(function (argv) {
+			// Set app-specific env for nightwatch session
+
+			// If possible, argv inputs and environment variables will be merged together
+			// If not, argv inputs will override environment variables.
+			if (argv.test_paths) {
+				// If argv.test_paths is set, add these paths to the environment variable
+				process.env.KNE_TEST_PATHS += ',' + argv.test_paths;
+			} else if (!process.env.KNE_TEST_PATHS) {
+				// If neither argv.test_paths nor the environment variable is set, throw an error.
+				var err = new Error('No test paths provided. Either set the --test_paths config option or the KNE_TEST_PATHS environment variable');
+				done(err);
+			}
+			process.env.KNE_SELENIUM_SERVER = selenium.path;
```

Comment:
since this is an env variable which may for whatever reason be set from the outside, perhaps do:
```
process.env.KNE_SELENIUM_SERVER = process.env.KNE_SELENIUM_SERVER || selenium.path;
```